openedx/core/djangoapps/user_authn/views/tests/test_reset_password.py

```diff
@@ -713,3 +713,87 @@ def test_reset_password_token_with_other_user(self):
 
         self.user = User.objects.get(pk=self.user.pk)
         self.assertFalse(self.user.is_active)
+
+
+@ddt.ddt
+@unittest.skipUnless(
+    settings.ROOT_URLCONF == "lms.urls",
+    "reset password tests should only run in LMS"
+)
+class ResetPasswordAPITest(CacheIsolationTestCase):
+    """Tests of the logistration API's password reset endpoint. """
+    request_factory = RequestFactory()
+    ENABLED_CACHES = ['default']
+
+    def setUp(self):
+        super(ResetPasswordAPITest, self).setUp()
+        self.user = UserFactory.create()
+        self.user.save()
+        self.token = default_token_generator.make_token(self.user)
+        self.uidb36 = int_to_base36(self.user.id)
+
+    @ddt.data(
+        (None, None, True),
+        (None, 'invalid_token', False),
+    )
+    @ddt.unpack
+    def test_invalid_token_in_password_reset_request(self, uidb36, token, status):
+        """
+         Test that user should not be able to reset password through invalid token
+        """
+        if uidb36 is None:
```

Comment:
Instead of these, you could directly do this:
```
kwargs={"uidb36": uidb36 or self.uidb36, "token": token or self.token}
```
OR
```
uidb36 = uidb36 or self.uidb36
token = token or self.token
```